benchmarks/lib/query_benchmark_common.dart

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2018, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:benchmark_harness/benchmark_harness.dart';
+
+/// `benchmark_harness-2` reports 10x the time the benchmarked code takes. This
+/// overrides the `measure` method to divide the result by 10.
```

Comment:
Perhaps leave a link to the issue explaining why you are doing this.

Reply:
> Perhaps leave a link to the issue explaining why you are doing this.

Done.

I wonder if we should use this class in all benchmarks, because sooner or later https://github.com/dart-lang/benchmark_harness/issues/30 will be fixed (I hope?), and when that's done we will have to update the other benchmarks (those that don't use this class) to report 10x numbers, to be compatible with old results stored in Golem.

I will update the PR again to do this.

Reply:
Updated PR to report accurate numbers in all benchmarks by dividing the benchmark_harness results by 10 before reporting.